dashboard/app/helpers/view_options_helper.rb

```diff
@@ -25,7 +25,7 @@ module ViewOptionsHelper
     :server_project_level_id,
     :game_display_name,
     :script_name,
-    :stage_position,
+    :lesson_position,
```

Comment:
I'm not sure what this does and if it should be changed or not

Reply:
There is a way to avoid changing the lesson_position stuff but it would mean leaving the routes file in a much messier place. I think it would be better to try to move toward lesson_position. Did you have any suggestions on where to break the chain?

Reply:
I think you responded to this somewhere else saying that you have updated the client code to use lessonPosition. This sounds great. 👍